lib/components/first-time-point/style.less

```diff
@@ -5,15 +5,17 @@
 
   @{prefix}arrow {
     left: -45px;
-    position: relative;
+    position: absolute;
+    top: -62px;
   }
 
   @{prefix}arrow-pointer {
-    width: 100px;
-    height: 60px;
+    width: 88px;
+    height: 37px;
```

Comment:
right on! does this reduce the footprint of the ftpo?

Reply:
Sure does!